bson/src/main/org/bson/codecs/IterableCodecProvider.java

```diff
@@ -27,7 +27,9 @@
  * A {@code CodecProvider} for classes than implement the {@code Iterable} interface.
  *
  * @since 3.3
+ * @deprecated Prefer {@link CollectionCodecProvider}
  */
+@Deprecated
```

Comment:
As we need this for our own driver code - should it actually be deprecated?

Reply:
Maybe not.  I'll think about it more.

Reply:
Done: removed deprecation.